socorro/scripts/fetch_crash_data.py

```diff
@@ -196,6 +207,10 @@ def main(argv=None):
     else:
         args = parser.parse_args(argv)
 
+    if args.fetchdumps and not args.fetchraw:
+        print("You cannot fetch dumps without also fetching the raw crash. Exiting.")
+        return 1
+
```

Comment:
The raw crash has a field that tells us what dumps came with the crash report. So if the user wants to download dumps, but not the raw crash, we can't really do that since without the raw crash, we have no idea what dumps exist.